src/components/map/mapControls/MapLayersControl.tsx

```diff
@@ -33,28 +36,78 @@ export default class MapLayersControl extends React.Component
         });
     }
 
+    public toggleTransitType = (type: TransitType) => {
+        NetworkStore.toggleTransitType(type);
+    }
+
+    public toggleIsLinksVisible = () => {
```

Comment:
These names feel like bad english 😅 
Could they just be `toggleLinksVisibility` etc

Reply:
I'm also not sure why these are wrapped in local functions when the ones from networkstore are selfexplanatory already..